tools/topology/m4/tdfb_defines.m4

```diff
@@ -0,0 +1,6 @@
+define(DEF_TDFB_PRIV, concat(`tdfb_priv_', PIPELINE_ID))
+define(DEF_TDFB_BYTES, concat(`tdfb_bytes_', PIPELINE_ID))
+define(DEF_TDFB_ONOFF_ENUM, concat(`tdfb_enable_', PIPELINE_ID))
+define(DEF_TDFB_DIRECTION_ENUM, concat(`tdfb_azimuth_', PIPELINE_ID))
```

Comment:
Also here the control could be called just e.g. name='TDFB10.0 azimuth'. There might later a control for elevation, so I thought calling out azimuth and elevation could be useful. Just "angle" or "direction" would not be exact. Thoughts?